application/src/main/resources/db/V11__Add_Help_Channel_Plus_Creator.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE add_help_channel
```

Comment:
i dont like the concept of this table. i would find it better if it would be a general purpose help channel list.

hence a better naming would be `help_channels` or `help_threads`

Reply:
I agree, struggled reading some things because of the constant "ADD"